File: cjxmodel/src/main/java/com/model/cjx/common/MyLog.java
package com.model.cjx.common;

import android.util.Log;

import com.model.cjx.BuildConfig;

/**
 * create by cjx on 2018/12/17
 */
public class MyLog {
    public static void E(String msg){
        if(BuildConfig.DEBUG){
            Log.e("cjx", msg);
        }
    }
}

File: cjxmodel/src/main/java/com/model/cjx/common/MyLog.kt
package com.model.cjx.common

import android.util.Log

import com.model.cjx.MyApplication

/**
 * Created by cjx on 17-8-7.
 * 打印日志, release版本不打印
 */
object MyLog {

    fun e(msg: String, tag: String = "cjx") {
        if (MyApplication.instance.debug) {
            Log.e(tag, msg)
        }
    }

    fun e(msg: String){
        if (MyApplication.instance.debug) {
            Log.e("cjx", msg)
        }
    }
}
